ampersand-form-view.js

```diff
@@ -7,6 +7,7 @@ var result = require('amp-result');
 
 function FormView(opts) {
     opts = opts || {};
+    opts.autoRender = opts.autoRender !== undefined ? opts.autoRender : true;
```

Comment:
on second thought, all of the other bool-ish values expect true boolean
inputs.  I will fall in line for consistency and simplicity.  updated.

On Mon, Mar 16, 2015 at 8:43 AM, Michael Garvin <notifications@github.com>
wrote:

> In ampersand-form-view.js
> <https://github.com/AmpersandJS/ampersand-form-view/pull/35#discussion_r26496617>
> :
>
> > @@ -7,6 +7,7 @@ var result = require('amp-result');
> >
> >  function FormView(opts) {
> >      opts = opts || {};
> > +    opts.autoRender = opts.autoRender !== undefined ? opts.autoRender : true;
>
> Still, that line is a little noodley, it wouldn't hurt to change from a
> ternary assignment to an if statement.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AmpersandJS/ampersand-form-view/pull/35/files#r26496617>
> .
>



-- 
Chris Dieringer
p: (503)358-8577
e: cdieringer@mrn.org
w: http://cdaringe.net/